test/vl-template.test.html

```diff
@@ -13,28 +16,37 @@
   <test-fixture id="vl-template-fixture">
     <template>
       <vl-template>
-        <span slot="header">header-content</span>
+        <vl-header slot="hader" data-vl-identifier="59188ff6-662b-45b9-b23a-964ad48c2bfb"></vl-header>
         <span slot="main">main-content</span>
-        <span slot="footer">footer-content</span>
+        <vl-footer slot="footer" data-vl-identifier="0337f8dc-3266-4e7a-8f4a-95fd65189e5b"></vl-footer>
       </vl-template>
     </template>
   </test-fixture>
 
-<script>
-  suite('vl-template', () => {
-    test('heeft de correcte slot contents', () => {
-      const element = fixture('vl-template-fixture');
-      const headerElement = element.shadowRoot.querySelector('vl-header > slot[name="header"]').assignedElements()[0];
-      const mainElement = element.shadowRoot.querySelector('div.vl-page > main.vl-main-content > slot[name="main"]').assignedElements()[0];
-      const footerElement = element.shadowRoot.querySelector('vl-footer > slot[name="footer"]').assignedElements()[0];
-
-      assert.isTrue(headerElement.innerText === 'header-content');
-      assert.isTrue(mainElement.innerText === 'main-content');
-      assert.isTrue(footerElement.innerText === 'footer-content');
+  <script>
+    suite('vl-template', () => {
+      const should = chai.should();
+
+      test('bevat een vl-header', () => {
+        const element = fixture('vl-template-fixture');
+        const header = element.shadowRoot.querySelector('slot[name="header"]');
+        should.exist(header);
+      });
```

Comment:
heeft da nut om da hier te testen? ja, tuurlijk bevat DEZE vl-template een vl-footer en header, want ge hebt die er zelf ingestoken, maar das toch geen functionaliteit van de vl-template?

Reply:
Het is niet omdat je er een slot in steekt dat de wegcomponent een slot voorzien heeft waar dit gerenderd wordt, dus unit test is nodig lijkt me.